src/utils/request.ts

```diff
@@ -43,3 +45,16 @@ export function assertMethod (event: CompatibilityEvent, expected: HTTPMethod |
     })
   }
 }
+
+export function getHeaders (event: CompatibilityEvent): IncomingHttpHeaders {
+  return event instanceof IncomingMessage
```

Comment:
CompatibilityEvent always has `.req` and `.res` props. We do not need to handle Node.js `req` directly being passed.